docs/validation.md

```
@@ -0,0 +1,113 @@
+    "email": [
+      "Email is required.",
+      "Email must be a valid email address."
+    ],
+    "password": [
+      "Password must be at least 8 characters long."
+    ]
+  }
+}
+```
+
+## Project Management
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)